src/pages/Register.jsx

```diff
@@ -56,41 +60,66 @@ export default function Register() {
   }
 
   return (
-    <div className="m-auto w-4/6">
-      <Card className="p-10">
-        <h1 className="text-center mb-5 font-bold text-3xl">Register</h1>
-        <form className="grid gap-y-5">
-          <TextField
-            value={email}
-            onChange={(e) => setEmail(e.target.value)}
-            type="email"
-            placeholder="Enter your email"
-          />
-          <TextField
-            value={password}
-            onChange={(e) => setPassword(e.target.value)}
-            type="password"
-            placeholder="Enter your password"
-          />
-          <TextField
-            value={password_confirmation}
-            onChange={(e) => setPassword_confirmation(e.target.value)}
-            type="password"
-            placeholder="Enter your password confirmation"
-          />
-          <div className="flex justify-center">
-            <Button onClick={() => handleSubmit()} variant="contained">
-              Register
-            </Button>
-          </div>
-        </form>
-        <small className="mt-5">
-          Already have an account?{" "}
-          <Link className="underline" to={"/login"}>
-            Login
-          </Link>
-        </small>
-      </Card>
+    <div className="absolute flex w-full h-full bg-gray-300">
+      <div className="m-auto w-5/6 md:w-4/6">
+        <Card className="p-6">
+          <h1 className="text-center mb-5 font-bold text-3xl">Register</h1>
+          <form className="grid gap-y-5">
+            <input
+              className="rounded-full border p-2 outline-none"
+              value={firstName}
```

Comment:
First name and last name are not submitable on devise register. I might just remove them all together from the model to be honest, I think we only need a page to change the email address and password